addons/web_editor/static/src/js/wysiwyg/widgets/color_palette.js

```diff
@@ -100,8 +100,8 @@ export class ColorPalette extends Component {
             this.$ = $el.find.bind($el);
 
             $el.on('click', '.o_we_color_btn', this._onColorButtonClick.bind(this));
-            $el.on('mouseenter', '.o_we_color_btn', this._onColorButtonEnter.bind(this));
-            $el.on('mouseleave', '.o_we_color_btn', this._onColorButtonLeave.bind(this));
+            $el.on('mouseenter', '.o_we_color_btn .o_we_color_btn:not(.o_custom_gradient_btn):not(.selected)', this._onColorButtonEnter.bind(this));
+            $el.on('mouseleave', '.o_we_color_btn .o_we_color_btn:not(.o_custom_gradient_btn):not(.selected)', this._onColorButtonLeave.bind(this));
```

Comment:
This is not enough. You will also have to add this for `.o_custom_gradient_editor we-button` for `.active`.

```js
$el.on('mouseenter', '.o_custom_gradient_editor we-button:not(.o_remove_color)', this._onGradientButtonEnter.bind(this));
$el.on('mouseleave', '.o_custom_gradient_editor we-button:not(.o_remove_color)', this._onGradientButtonLeave.bind(this));
```
to
```js
$el.on('mouseenter', '.o_custom_gradient_editor we-button:not(.o_remove_color):not(.active)', this._onGradientButtonEnter.bind(this));
$el.on('mouseleave', '.o_custom_gradient_editor we-button:not(.o_remove_color):not(.active)', this._onGradientButtonLeave.bind(this));
```